pallets/issue/src/lib.rs

```diff
@@ -147,6 +152,25 @@ pub mod pallet {
 	pub(super) type IssueMinimumTransferAmount<T: Config> =
 		StorageValue<_, BalanceOf<T>, ValueQuery>;
 
+	#[pallet::storage]
+	pub(super) type LimitVolumeAmount<T: Config> =
+		StorageValue<_, Option<BalanceOf<T>>, ValueQuery>;
+
+	///Currency id for limit rate volume value. DOT, BTC or PEN
```

Comment:
I would not mention BTC here as we don't use BTC. I would change it to 
```
/// CurrencyID that represents the currency in which the volume limit is measured, eg DOT, USDC or PEN
```

Reply:
Done